src/java.xml/share/classes/jdk/xml/internal/Utils.java

```diff
@@ -41,14 +41,9 @@ public class Utils {
      * and processors
      */
     static {
-        try {
-            String val = SecuritySupport.getSystemProperty("jaxp.debug");
-            // Allow simply setting the prop to turn on debug
-            debug = val != null && !"false".equals(val);
-        }
-        catch (SecurityException se) {
-            debug = false;
-        }
+        String val = SecuritySupport.getSystemProperty("jaxp.debug");
+        // Allow simply setting the prop to turn on debug
+        debug = val != null && !"false".equals(val);
```

Comment:
It might be that java.xml takes several passes to clean up but I assume the SecuritySupport.getSystemProperty methods can be removed and use-sites changed to use System.getProperty.

Reply:
I'll see what I can do with a bulk, find/replace.